es/es-dashboards/README.md

```diff
@@ -15,44 +15,63 @@ kibana.index: ".dashboards"
 
 Adapt if needed ```elasticsearch.url``` and ```server.host```.
 
-
-
 Start Kibana manually:
 
 ```
 cd kibana/bin
 ./kibana
 ```
 
-### Maven installation
+## Maven installation
 
-Maven can take care of the installation steps:
-* download
-* initialize collection
-* start
+1. Maven can take care of the installation steps:
 
-Use the following commands:
+   * download
+   * initialize collection
+   * start
 
-```
-cd es/es-dashboard
-mvn install -Pkb-download
-mvn exec:exec -Dkb-start
-```
+2. Use maven to download:
 
-### Import Configuration
+   ```
+   cd es/es-dashboard
+   mvn install -Pkb-download
+   ```
 
-Kibana should be running from:
+3. Run locally:
 
-```
-http://localhost:5601
+   ```
+   mvn exec:exec -Dkb-start
+   ```
 
-```
- and should be visible within the geonetwork interface at:
+## Docker compose installation
+
+1. Use docer compose with the provided [docker-compose.yml](docker-compose.yml):
```

Comment:
docer > docker

Reply:
If you use GitHub to suggest changes for minor typos it can greatly speed up the review process

Reply:
aside: I not this has already been merged and the branch deleted, so a new pull request will be needed.